python/lsst/sims/maf/maps/createStarDensitymap.py

```diff
@@ -13,100 +13,102 @@
 
 # Connect to fatboy with: ssh -L 51433:fatboy-private.phys.washington.edu:1433 gateway.astro.washington.edu
```

Comment:
should this be ```simsuser@gateway.astro.washington.edu``` (since most users will not have their own ```@astro``` account, and will need to be logging in through the communal ```simsuser``` account)?